packages/flutter/test/material/menu_anchor_test.dart

```diff
@@ -2532,6 +2532,22 @@ void main() {
           ..rect(color: overlayColor.withOpacity(0.1)),
       );
     });
+
+    testWidgets('MenuItemButton can build when its child is null', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        const MaterialApp(
+          home: Scaffold(
+            body: SizedBox(
+              width: 200,
+              child: MenuItemButton(),
+            ),
+          ),
+        ),
+      );
+
+      // exception `Null check operator used on a null value` would be thrown.
+      expect(tester.takeException(), isNull);
```

Comment:
Maybe check the type of exception that is thrown?

```suggestion
      expect(tester.takeException(), isA(TypeError));
```

Reply:
My understanding is that this test expects an empty `MenuItemButton()` constructor to build without errors.

<br>

Kinda similar to one of the date picker tests:

https://github.com/flutter/flutter/blob/5ecd4ea83d66e0aa8ffef205f4c2732f9a5818e7/packages/flutter/test/cupertino/date_picker_test.dart#L127-L137

Reply:
Oh, duh.  Sorry, misread that. Ignore my comment.